matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java

```diff
@@ -0,0 +1,469 @@
+                generator.newReactiveCapabilityCurve()
+                        .beginPoint()
+                        .setP(mGen.getPc1())
+                        .setMaxQ(mGen.getQc1Max())
+                        .setMinQ(mGen.getQc1Min())
+                        .endPoint()
+                        .beginPoint()
+                        .setP(mGen.getPc2())
+                        .setMaxQ(mGen.getQc2Max())
+                        .setMinQ(mGen.getQc2Min())
+                        .endPoint()
+                        .add();
+            }
+        });
+    }
+
+    private static Bus createBus(MBus mBus, VoltageLevel voltageLevel) {
+        String busId = getBusId(mBus.getNumber());
+        LOGGER.debug("Creating bus {}", busId);
+        Bus bus = voltageLevel.getBusBreakerView().newBus()
+                .setId(busId)
+                .setName(busId)
+                .add();
+        bus.setV(mBus.getVoltageMagnitude() * voltageLevel.getNominalV())
+                .setAngle(mBus.getVoltageAngle());
+        return bus;
+    }
+
+    private static Substation createSubstation(Network network, String substationId) {
+        LOGGER.debug("Creating substation {}", substationId);
+        Substation substation = network.getSubstation(substationId);
+        if (substation == null) {
+            substation = network.newSubstation()
+                    .setId(substationId)
+                    .add();
+        }
+        return substation;
+    }
+
+    private static VoltageLevel createVoltageLevel(MBus mBus, String voltageLevelId, Substation substation, Network network, PerUnitContext perUnitContext) {
+        double nominalV = perUnitContext.isIgnoreBaseMva() || mBus.getBaseVoltage() == 0 ? 1 : mBus.getBaseVoltage();
+        VoltageLevel voltageLevel = network.getVoltageLevel(voltageLevelId);
+        if (voltageLevel == null) {
+            LOGGER.debug("Creating voltagelevel {}", voltageLevelId);
+            voltageLevel = substation.newVoltageLevel()
+                    .setId(voltageLevelId)
+                    .setNominalV(nominalV)
+                    .setTopologyKind(TopologyKind.BUS_BREAKER)
+                    .add();
+        }
+        return voltageLevel;
+    }
+
+    private static void createLoad(MBus mBus, VoltageLevel voltageLevel) {
+        if (mBus.getRealPowerDemand() != 0 || mBus.getReactivePowerDemand() != 0) {
+            String busId = getBusId(mBus.getNumber());
+            String loadId = busId + "-L";
+            LOGGER.debug("Creating load {}", loadId);
+            voltageLevel.newLoad()
+                .setId(loadId)
+                .setConnectableBus(busId)
+                .setBus(busId)
+                .setP0(mBus.getRealPowerDemand())
+                .setQ0(mBus.getReactivePowerDemand())
+                .add();
+        }
+    }
+
+    private static String createGenId(MGen mGen, Network network) {
+        String genIdPrefix = "G" + mGen.getNumber() + "-";
```

Comment:
The naming convention is different for generator: for generator you use a prefix, but for loads and shunts you use a post-fix

Reply:
fixed: used a prefix, consistently